spec/game_team_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "spec_helper"
+
+RSpec.describe GameTeam do
+  before(:each) do
+    gameteam_data = {
+        game_id: "2012030221",
+        team_id: "3",
+        hoa: "away",
+        result: "LOSS",
+        settled_in: "OT",
+        head_coach: "John Tortorella",
+        goals: 2,
+        shots: 8,
+        tackles: 44,
+        pim: 8,
+        ppo: 3,
+        ppg: 0,
+        faceoff_win_per: 44.8,
+        giveaways: 17,
+        takeaways: 7
+    }
+    @gameteam_attr_test = GameTeam.new(gameteam_data)
```

Comment:
great naming, I like that it is really clear and know exactly what it's for in this line

Reply:
ah nice, I was confused at first "what's a attr_test? is that like a attr_accessor that I've never used?" then realized it's just a test variable. Makes sense!